File: 2708.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');
let cont = 0;
let jipes = 0;

while(true){
  let [a, b] = lines.shift().trim().split(" ");

  if(a == "ABEND"){
    break;
  }

  if(a == "SALIDA"){
    cont += parseInt(b);
    jipes += 1;
  }
  else if(a == "VUELTA"){
    cont -= parseInt(b);
    jipes -= 1;
  }
}

console.log(cont);
console.log(jipes);

File: 2708.java
import java.io.IOException;
import java.util.Scanner;

public class Main {

    public static void main(String[] args) throws IOException {
    	Scanner leitor = new Scanner(System.in);
    	int turistas = 0;
    	int jipes = 0;
    	while (true) {
    		String input = leitor.next();
    		if (input.equals("SALIDA")) {
    			jipes++;
    			turistas += leitor.nextInt();
    		}
    		else if (input.equals("VUELTA")) {
    			jipes--;
    			turistas -= leitor.nextInt();
    		} else {
    			break;
    		}
    	}
    	System.out.println(turistas);
    	System.out.println(jipes);
    }
    
}
